packages/graphile-build-pg/src/plugins/introspectionQuery.js

```diff
@@ -157,6 +158,7 @@ with
       nullif(att.atttypmod, -1) as "typeModifier",
       att.attnotnull as "isNotNull",
       att.atthasdef as "hasDefault",
+      ${serverVersionNum >= 100000 ? `att.attidentity as "identity",` : ``}
```

Comment:
Actually I think it might be best to *always* define the columns rather than making `void` an option; so this should be something like:

```
${serverVersionNum >= 100000 ? 'att.attidentity' : "''"} as "identity",
```

This also makes the expected column names easier to scan.